test/solidity/block/BlockInfo.js

```diff
@@ -0,0 +1,105 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+const { BigNumber } = require('ethers');
+const Constants = require('../../constants')
+
+describe('@solidityequiv1 BlockInfo Test Suite', function () {
+  let blockInfo, provider, signers
+
+  before(async function () {
+    signers = await ethers.getSigners()
+    provider = ethers.getDefaultProvider();
+    
+    const factory = await ethers.getContractFactory(Constants.Path.BLOCK_INFO)
+    blockInfo = await factory. deploy({ gasLimit: 15000000 })
+  })
+
+  // EIP-1559 does not apply to Hedera
+  it('should be able to execute getBlockBaseFee()', async function () {
+    const blockBaseFee = await blockInfo.getBlockBaseFee()
+    expect(blockBaseFee).to.equal(0)
+  })
+
+  // Turn off until mirror node issue is resolved: https://github.com/hashgraph/hedera-mirror-node/issues/7045
+  xit('should be able to get the hash of a given block when the block number is one of the 256 most recent blocks', async function () {
```

Comment:
Please turn test on. Instead of turning test off expect the incorrect value but you can leave the ticket issue here. Once that issue is fixed we should see this test start to fail.
This avoids where we forget to come back to fix it.

Same case for other tests in this case

Reply:
The test already fails.  The following error is returned : 
```
Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"from":"0x05FbA803Be258049A27B820088bab1cAD2058871","gasLimit":{"type":"BigNumber","hex":"0x89a758"},"to":"0xA4096Ee370402B804288a967dE244844287028AD","data":"0x9663f88f","accessList":null}, error={"name":"ProviderError","_stack":"ProviderError: [Request ID: 4218bfa1-39a1-45cb-9313-79fc59bd9ad2] execution reverted: Expected 32 bytes but got 48\n    at HttpProvider.request (/Users/user/projects/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at EthersProviderWrapper.send (/Users/user/projects/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)","code":-32008,"_isProviderError":true,"data":""}, code=CALL_EXCEPTION, version=providers/5.7.2)
```

Reply:
Turned on for now.

Reply:
Handles expected call exception.